api/types/network/endpoint.go

```diff
@@ -14,6 +14,9 @@ type EndpointSettings struct {
 	IPAMConfig *EndpointIPAMConfig
 	Links      []string
 	Aliases    []string // Aliases holds the list of extra, user-specified DNS names for this endpoint.
+	// MacAddress may contain a configured value from the API. Its value is copied
+	// into DesiredMacAddress when the container is created, after which MacAddress
+	// becomes operational data (it may contain a generated address).
```

Comment:
Ok, I'll change it to ...

```
	// MacAddress may be used to specify a MAC address when the container is created.
	// Once the container is running, it becomes operational data (it may contain a
	// generated address).
```